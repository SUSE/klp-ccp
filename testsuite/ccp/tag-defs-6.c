enum A
{
	a_a,
};

void pu_f(void)
{
	a_a;
}
