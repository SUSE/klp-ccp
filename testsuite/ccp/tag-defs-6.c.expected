enum A
{
	a_a,
};

void klpp_pu_f(void)
{
	a_a;
}
